dev/sparktestsupport/modules.py

```diff
@@ -431,6 +431,7 @@ def __hash__(self):
         "pyspark.ml.linalg.__init__",
         "pyspark.ml.recommendation",
         "pyspark.ml.regression",
+        "pyspark.ml.stat",
```

Comment:
We just took it out in https://github.com/apache/spark/commit/314cf51ded52834cfbaacf58d3d05a220965ca2a , but since this is adding back in ml.stat we also need to update setup.py (you might need to update your branch from the latest master to see this).